File: java/kata/src/main/java/necklaceRefactoringKata/JewelleryBox.java
package necklaceRefactoringKata;

import necklaceRefactoringKata.jewellery.Jewellery;
import necklaceRefactoringKata.jewellery.Ring;

import java.util.ArrayList;
import java.util.List;

public class JewelleryBox {
    public final List<Ring> ringCompartment;
    public final List<Jewellery> topShelf;
    public final List<Jewellery> mainSection;

    public JewelleryBox() {
        this.ringCompartment = new ArrayList<>();
        this.topShelf = new ArrayList<>();
        this.mainSection = new ArrayList<>();
    }
}

File: kotlin/src/main/kotlin/org/sammancoaching/necklace/JewelleryBox.kt
package org.sammancoaching.necklace

data class JewelleryBox(
  val ringCompartment: MutableList<Ring> = mutableListOf(),
  val topShelf: MutableList<Jewellery> = mutableListOf(),
  val mainSection: MutableList<Jewellery> = mutableListOf(),
)
